src/components/schedule/CreateShift.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Col } from 'react-bootstrap';
+import styled from 'styled-components';
+import { BsFillCircleFill } from 'react-icons/bs';
+import { withRouter } from 'react-router-dom';
+import HeaderWithBackArrow from '../navigation/back-header';
+import { SubmitButton, CancelButton } from '../../styled-components/form-components';
+
+const PaddedDiv = styled.div`
+  padding: 0 5%;
+`;
+
+const CenterCol = styled(Col)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+class CreateShift extends React.Component {
```

Comment:
It's definitely not a requirement, because this code is all correct and works great. But most of this project will be written with functional components as opposed to these class components.

Reply:
Got it, I'll try to stick to functional components from now on. Out of curiosity, is there a specific reason we're not using class components?

Reply:
The main benefits of functional components is that it makes code easier to build, read, understand and debug. There are a lot of good articles on this topic that do a better job of explaining than I could. 

But it's not that our product can't use class components, its just that functional components are a slight preference of mine (and the community as a whole).